compiler/testData/debug/stepping/kt15259.kt

```diff
@@ -0,0 +1,24 @@
+// IGNORE_BACKEND: JVM_IR
+// IGNORE_BACKEND_FIR: JVM_IR
+// FILE: test.kt
+interface ObjectFace
+
+private fun makeFace() = object : ObjectFace {
+    //Breakpoint! from the Evaluate Expression test suite.
```

Comment:
I'd replace these "Breakpoint!" lines with empty lines just to reduce confusion (one could think that this is somehow handled in compiler tests, which it's not)

Reply:
Done!